platform/pwl.c

```diff
@@ -849,30 +861,30 @@ display_stop_keyboard_repeat (PwlDisplay *self)
 
 
 static void
-display_start_keyboard_repeat (PwlDisplay *self,
+seat_start_keyboard_repeat (PwlSeat    *self,
                                uint32_t    timestamp,
                                uint32_t    key,
                                uint32_t    state)
```

Comment:
Please realign the rest of function arguments to the opening parenthesis here, too.

Reply:
done